docs/UserGuide.md

```diff
@@ -551,12 +546,6 @@ Example:
 
 1. `help` results in a popup window which contains all the commands in PROperty for easy reference during use.
 
-Visual example of correct output:
-
-![HelpCommandShowcase.png](images/user-guide-images/HelpMessageShowcase.png)
-![help message](images/user-guide-images/HelpMessageShowcase.png)
-![help message](images/user-guide-images/HelpMessageShowcase2.png)
-
```

Comment:
I think we should remove these unused images :)

Reply:
Alright sure! Will get to t.